packages/webpack-pwa-manifest-plugin/yarn.lock

```diff
@@ -0,0 +1,4969 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think because we use workspaces, there should only be one yarn.lock file, at the root of the repo. Can you remove this an re-run `yarn`?